i18n-auto-translate.config.json

```diff
@@ -31,6 +31,12 @@
                 "cl.json",
                 "ec.json"
             ]
+        },
+        {
+            "name": "no",
+            "files": [
+                "no.json"
+            ]
```

Comment:
Added "no" to the example because it is called "nb" in bing and libreTranslate and called "no" on Google Translate. So is a good example for the features implemented on this PR!